src/services/userFeedback/controller.ts

```diff
@@ -0,0 +1,46 @@
+import { StatusCodes } from 'http-status-codes';
+
+import { FastifyPluginAsync } from 'fastify';
+
+import { USER_FEEDBACK_RECEIVER_EMAIL } from '../../utils/config';
+import { userFeedbackSchema } from './fluent-schema';
+import { USER_FEEDBACK_MAIL } from './utils';
+
+const plugin: FastifyPluginAsync = async (fastify) => {
+  fastify.post<{ Body: { name: string; email: string; details: string } }>(
+    '/',
+    {
+      schema: userFeedbackSchema,
+      preHandler: fastify.verifyAuthentication,
+    },
+    async (request, reply) => {
+      const { member, body: data, hostname } = request;
+      const { log, mailer } = fastify;
+      console.log(hostname);
+      const subject = `User Feedback`;
```

Comment:
Question: Does Sentry provide the issue number in the payload ? In case they do, it would be interesting to have it in order to ease the search of the issue later in the emails.